cfg/examples/replication/dex-net_4.0_fc_pj.yaml

```diff
@@ -0,0 +1,36 @@
+policy:
+  type: fully_conv_pj
+
+  sampling_method: top_k
+  num_depth_bins: 16
+  gripper_width: 0.05
+  gqcnn_stride: 4
+  gqcnn_recep_h: 96
+  gqcnn_recep_w: 96
+
+  # filtering params
+  max_grasps_to_filter: 50
+  filter_grasps: 0
+
+  # metrics
+  metric:
+    type: fcgqcnn
+    gqcnn_model: models/FCGQCNN-4.0-PJ
```

Comment:
How does one download this model?

Reply:
Once again waiting for Box upload to update download shell scripts for the FC models/datasets.